src/other/index.jl

```diff
@@ -135,14 +135,16 @@ end
 function Base.getindex(x::AbstractIndex, idx::AbstractVector{Union{Bool, Missing}})
     if any(ismissing, idx)
         # TODO: this line should be changed to throw an error after deprecation
+        # throw(ArgumentError("missing values for column indexing are not alloved"))
```

Comment:
"allowed". I'd also put "for column indexing" at the end. Same below.

Reply:
fixed